spec/features/heroku_spec.rb

```diff
@@ -1,58 +1,73 @@
 require "spec_helper"
 
 RSpec.describe "Heroku" do
-  before do
-    drop_dummy_database
-    remove_project_directory
-  end
+  context "--heroku" do
+    before(:all) do
```

Comment:
The reason is the same of why this was done for the `new_project_spec`, and that is that running suspenders is slow, and thus we want to run it as few times as possibles. The rationale is that for each different configuration of the command we run it once and only once.

Since all the tests are doing is read only, it should never be too much of an issue, I think.